docs/development.md

```diff
@@ -80,8 +80,7 @@ $ npm run dev
 > my-app@1.0.0 start /Users/z/Desktop/foo
 > probot run ./index.js
 
-Yay, the plugin was loaded!
-18:11:55.838Z DEBUG Probot: Loaded plugin: ./index.js
+18:11:55.838Z INFO probot: Yay, the app was loaded!
```

Comment:
This matches the [output of a new template](https://github.com/probot/template/blob/master/index.js)